install/kubernetes/cilium/values.yaml.tmpl

```diff
@@ -3179,3 +3179,15 @@ authentication:
       agentSocketPath: /run/spire/sockets/agent/agent.sock
       # -- SPIRE connection timeout
       connectionTimeout: 30s
+
+# -- Configure the client side rate limit for the agent and operator
+#
+# If the amount of requests to the Kubernetes API server exceeds the configured
+# rate limit, the agent and operator will start to throttle requests by delaying
+# them until there is budget or the request times out.
+clientRatelimit:
```

Comment:
We already have K8s related options like `K8sServiceHost` at the top of the file. How about naming this `K8sClientRateLimit` and putting it next to them?

Reply:
Yea, that makes sense